assets/aml-benchmark/scripts/custom_dataset_preprocessors/base_preprocessor_template.py

```diff
@@ -0,0 +1,83 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ---------------------------------------------------------
+
+"""Base template for custopm preprocessor script."""
+
+from typing import Any, Dict, List, Union
+import argparse
+import json
+import logging as logger
+import pandas as pd
+
+
+def _parse_args():
+    """Parse the arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input_path", type=str, required=True)
+    parser.add_argument("--output_path", type=str, required=True)
+    argss = parser.parse_args()
```

Comment:
Typo: `args`

Reply:
I have actually used `argss` variable instead of `args`.